chcemvediet/apps/anonymization/models.py

```diff
@@ -0,0 +1,126 @@
+
+    # May NOT be NULL;
+    attachment = models.ForeignKey(Attachment)
+
+    # May NOT be NULL;
+    successful = models.BooleanField(default=False,
+        help_text=squeeze(u"""
+            Boolean denoting whether normalization has succeeded.
+            """))
+
+    # May be NULL;
+    file = models.FileField(upload_to=u'attachment_normalizations', max_length=255, null=True,
+            help_text=squeeze(u"""
+                It can be NULL if normalization failed or as a result of normalization no file 
+                was created.
+                """))
+
+    # May NOT be empty: Automatically adjust extension in save() when creating new object;
+    name = models.CharField(max_length=255,
+            help_text=squeeze(u"""
+                Attachment file name, e.g. "document.pdf". Automatically adjust extension when 
+                creating a new object based on content_type.
+                """))
+
+    # May be NULL; Explicitly set in save() when creating a new object.
+    content_type = models.CharField(max_length=255, null=True,
+            help_text=squeeze(u"""
+                Attachment content type, e.g. "application/pdf". Explicitly set when creating a 
+                new object. The value may be specified even if normalization failed.
+                """))
+
+    # May NOT be NULL; Automatically computed in save() when creating a new object if undefined.
+    created = models.DateTimeField(blank=True,
+            help_text=squeeze(u"""
+                Date and time the attachment was uploaded or received by an email. Leave blank 
+                for current time.
+                """))
+
+    # May be NULL; Automatically computed in save() when creating a new object.
+    size = models.IntegerField(null=True, blank=True,
+            help_text=squeeze(u"""
+                It can be NULL if file is NULL. Attachment file size in bytes. Automatically 
+                computed when creating a new object.
+                """))
+
+    # May NOT be NULL;
+    debug = models.TextField(blank=True,
+            help_text=squeeze(u"""
+                Debug message from normalization. May not be NULL, but it can be empty.
+                """))
+
+    # Backward relations added to other models:
+    #
+    #  -- Attachment.attachment_normalization_set
+    #     May be empty
+
+    # Indexes:
+    #  -- attachment:       ForeignKey
+
+    objects = AttachmentNormalizationQuerySet.as_manager()
+
+    @cached_property
+    def content(self):
+        if not self.file:
+            logger = logging.getLogger(u'chcemvediet.apps.anonymization')
+            logger.error(u'{}\'s file is NULL'.format(self))
+            return None
+
+        try:
+            self.file.open(u'rb')
+            return self.file.read()
+        except IOError:
+            logger = logging.getLogger(u'chcemvediet.apps.anonymization')
+            logger.error(u'{} is missing its file: "{}".'.format(self, self.file.name))
```

Comment:
Pls otestuj, ze errory poslane do loggera `chcemvediet.apps.anonymization` sa pri localhost testovani vypisu do konzoly a v produkcii zapisu do logov. Ci netreba nahodou upravit `LOGGING` v `settings`

Reply:
V mode [2] local development mi logovanie slo do konzoly. Ak som chcel nastavit na produkcny mod [7], tak mi vyhodilo error s SQL. Pokusim sa chybu vyriesit a otestovat.

Reply:
@viliambalaz Pravda, produkcny mod pouziva MySQL, kym lokalny mod iba SQLite (aby nebolo treba instalovat MySQL lokalne). Nemusis teda testovat produkcny mod, iba skopiruj nastavenia `LOGGING` do settingov pre lokalny mod a pozri ci tie settingy funguju a zaloguju `chcemvediet.apps.anonymization` niekam do suboru.

Reply:
Ano, loguje sa aj do suborov. Pre classu `poleno` sa vypisu dve chybove hlasky, (asi preto) lebo je vytvoreny logger pre `root` a aj pre `poleno`.
![log](https://user-images.githubusercontent.com/29918752/61527336-775e8480-aa1c-11e9-8927-d616bf42c90a.png)

Reply:
@viliambalaz Dvojnasobne logy su sposobene tym, ze tam chyba `u'propagate': False`. Ale nejdem to tam davat v tomto PR. Mam pocit, ze nieco blblo a ked bolo zakazane propagate, tak nieco sa obcas nezalogovalo. Ale uz nepamatam. Nateraz teda OK.